revolv/base/management/commands/autorepay.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from revolv.payments.models import AdminRepayment
+from revolv.project.models import Project
+
+
+class Command(BaseCommand):
+    option_list = BaseCommand.option_list + (
+        make_option('--override',
+                    action='store_true',
+                    dest='override',
+                    default=False,
+                    help='Forcibly run command on day other than 1st or 15th.'),
+    )
+
+    def handle(self, *args, **options):
+        """
+        Run this function from the command line with `python manage.py autorepay`.
+
+        This command iterates through all completed projects and makes a
+        repayments to each of them according to their respective
+        MonthlyReinvestableAmount and MonthlyOrganizationalCost.
+
+        A Repayment is only made towards a project if it hasn't already been
+        fully repayed.
+
+        This command is cron-ed to run on the 1st and 15th of every month. To
+        run it on any other day, the `--override` flag must be supplied.
+        """
+        day = timezone.now().day
+        if not any([day == 1, day == 15, options['override']]):
+            print >> sys.stderr, ("autorepay command can only be run on the 1st "
+                                  "or 15th of every month unless the `--override` "
+                                  "flag is supplied. use with caution")
+            sys.exit(1)
```

Comment:
I want `sys.exit(1)` because I'd like for the command to exit with a `FAIL` system code. afaik it's standard behavior for a unix command to exit with a bad exit code if it fails, and this is one of those cases imo